config/custom_items.txt

```diff
@@ -329,7 +329,7 @@ item_path:/obj/item/clothing/accessory/badge/holo/detective/ruda
 {
 ckey: interrolouis
 character_name: Kai Highlands
-/obj/item/borg/upgrade/modkit/chassis_mod/kai
+item_path:/obj/item/borg/upgrade/modkit/chassis_mod/kai
```

Comment:
Missing the space

Reply:
I went ahead and left the space out, since their other fluff item in the above lines didn't have a space inbetween it, either. I'll go ahead and add spaces to both of them.